identity/config.go

```diff
@@ -0,0 +1,20 @@
+package identity
+
+import "perun.network/go-perun/wallet"
+
+// WalletConfig defines wallet specific configuration parameters.
+type WalletConfig struct {
+	Addr         wallet.Address
+	KeystorePath string
+	Pwd          string //The passwords are received by the cli app as command line arguments are passed to the config.
```

Comment:
Just write what it is, not how it is getting there. Also; `Pwd` on Unix normally means working directory.  
In the future this should probably not be a string, but the last time i searched for it, there was no good library for secure-strings in golang…

Reply:
removed the comment and updated the variable name to `Password`. 
Regarding secure way for handling passwords, can it be added as one the requirements in the [Identity Management task](#43) ?

Reply:
When it is a requirement from your side too, for sure. But i think it should go into #42 which is probably a sub-task of the identity managment.